kubernetes/helm/pinot/templates/controller/ingress.yaml

```diff
@@ -0,0 +1,30 @@
+{{- if .Values.controller.ingress.enabled -}}
+{{- $ingressPath := .Values.controller.ingress.path -}}
+{{- $serviceName := include "pinot.controller.fullname" . -}}
+{{- $servicePort := .Values.controller.service.port -}}
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: {{ include "pinot.controller.fullname" . }}
+{{- if .Values.controller.ingress.annotations }}
+  annotations:
+{{ toYaml .Values.controller.ingress.annotations | indent 4 }}
+  {{- end }}
+  labels:
+    {{- include "pinot.controllerLabels" . | nindent 4 }}
+spec:
+{{- if .Values.controller.ingress.tls }}
+  tls:
+{{ toYaml .Values.controller.ingress.tls | indent 4 }}
+{{- end }}
+  rules:
+    {{- range .Values.controller.ingress.hosts }}
+    - host: {{ . }}
+      http:
+        paths:
+          - path: {{ $ingressPath }}
+            backend:
+              serviceName: {{ $serviceName }}
```

Comment:
What's the right structure, ref: https://kubernetes.io/docs/concepts/services-networking/ingress
```
backend:
          service:
            name: service1
            port:
              number: 80
```

Reply:
@dianaarnos This is exactly why I suggested asking about the minimum supported version of Kubernetes. The version we build together is an older version then is now the default.

Reply:
I asked on Slack, but didn't get an answer, so we tried it our own way and wait for the comments.

Reply:
That's perfectly fine, in retrospect my comment probably was a bit harsher than intended. Sorry about that :(

Reply:
The structure followed on this PR is related to the `extensions/v1beta1` api, available on 1.18. As said here:  https://github.com/apache/pinot/pull/7997#discussion_r787729991